kong/plugins/response-transformer/body_transformer.lua

```diff
@@ -90,9 +93,9 @@ end
 
 
 function _M.transform_json_body(conf, buffered_data)
-  local json_body = read_json_body(buffered_data)
+  local json_body = parse_json(buffered_data)
   if json_body == nil then
-    return
+    return nil
```

Comment:
```suggestion
    return nil, "failed parsing json body"
```
I think it would be useful to return a warning, then log it  in case of parsing failure